src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -159,13 +160,20 @@ export class SimpleFilter extends Component {
           return 'score';
         }
       }
-    })
+    }),
+    /**
+     * Specifies whether to show a button to clear all selected values.
+     *
+     * Default value is `false`.
```

Comment:
You can remove this line; TypeDoc will automatically generate a tag indicating what the default value is.